src/common/constants.ts

```diff
@@ -31,3 +33,36 @@ export const SIP_010 = {
     trait: 'ft-trait',
   },
 };
+
+export interface Network {
+  url: string;
+  name: string;
+  chainId: ChainID;
+}
+
+export interface Networks {
+  [key: string]: Network;
+}
```

Comment:
```suggestion
export type Networks = Record<string, Network>;
```
Think it's encouraged to use the Record type util